Project.toml

```diff
@@ -62,7 +62,7 @@ RRTMGP = "0.19"
 SciMLBase = "2.12"
 StaticArrays = "1.7"
 Statistics = "1"
-SurfaceFluxes = "0.11, 0.12"
+SurfaceFluxes = "0.12"
```

Comment:
What changes are needed to support 0.11 vs 0.12?

Can we support both?

Reply:
We have changed the definitions of sensible and latent heat fluxes following the dycore paper equations (https://github.com/CliMA/SurfaceFluxes.jl/pull/176/files).

Technically we can support both, because the API is not changed. However the definitions of what SHF and LHF mean are different, so I was worried that supporting both will cause more problems. 

But lets keep v0.11 for now, in case we need to go back to previous model versions.

Reply:
Decided to keep both